src/sql/src/plan/statement/ddl.rs

```diff
@@ -289,6 +290,14 @@ pub fn plan_create_table(
     } else {
         scx.allocate_qualified_name(normalize::unresolved_object_name(name.to_owned())?)?
     };
+
+    if scx.item_exists(&name) {
+        return Err(PlanError::ItemAlreadyExists {
+            name: name.item,
+            item_type: CatalogItemType::Table,
```

Comment:
I think what we want here is the type of the item that already exists, not the type of the item that we're trying to create. The existing item is not necessarily a table.

Reply:
Same for the other places this is initialized.